created the become a partner page (#260)
mautic/mautic-community-handbook: docs/policies/become_a_community_partner.rst

Reviewer: How to become a Mautic community partner

.. vale off 

How to become a mautic community partner
########################################

.. vale on

The Mautic Community Partners Program showcases organizations that are contributing to Mautic - both financially and practically - so that members of the community can find service providers who are :xref:`Makers rather than takers` to work with. These organizations represent the ideals of providing ongoing contributions that open source software rely upon.

How do organizations become a partner?
======================================

There are two elements to consider when reviewing an application to join the Partners Programme, both relate to whether the organization is supporting the community.

Financial contribution
----------------------

> 

Reviewer: ```suggestion
It's required that all Community Partners are financially supporting the growth of the Mautic community as a :xref:`Corporate Member of Mautic`. Members pay for all memberships annually, and via :xref:`Open Collective`. Prior to v2 of the Mautic governance model which came into effect on ``12th December 2023``, companies who sponsored Mautic at $100 per month or more were also eligible. Companies who were already sponsoring at this tier when the new governance model came into effect and continue to do so are 'grandfathered' in, providing that their sponsorship continues at this tier, for a period of 12 months. This gives them until ``12th December 2024`` to transition to becoming a Corporate Member to maintain their status as a Mautic Partner.
```

It's required that all Community Partners are financially supporting the growth of the Mautic community as a :xref:`Corporate Member of Mautic`. Members pay for all memberships annually, and via our :xref:`Open Collective`. Prior to v2 of the Mautic governance model which came into effect on ``12th December 2023``, companies who sponsored Mautic at $100 per month or more were also eligible. Companies who were already sponsoring at this tier when the new governance model came into effect and continue to do so are 'grandfathered' in, providing that their sponsorship continues at this tier, for a period of 12 months. This gives them until ``12th December 2024`` to transition to becoming a Corporate Member in order to maintain their status as a Mautic Partner.

> Practical Contribution

Reviewer: ```suggestion
Practical contribution
```

sentence-style capitalization

----------------------

> Of course money is not everything, and in fact contributing to Mautic in practical ways, whether that might be through code contributions, leading a team, organizing official events or meetups, contributing to the Documentation resources that we have or working on tasks in one of our five Community Teams are all vital to the growth of our community.

Reviewer: ```suggestion
Of course, money isn't everything. Contributing to Mautic in practical ways, whether through code contributions, leading a team, organizing events or meetups, improving documentation, or working on tasks within the Community Teams, is essential to the growth of our community.
```

More vale fixes

Author: @fakela @RCheesley I have made all the changes. Looking forward to to your review

For this reason, organizations who are wishing to become and maintain their status as a Mautic Community Partner must demonstrate regular, meaningful contributions to the Mautic project from their staff.

> We measure this in several ways including through the Community Dashboard, reviewing team activity, and engagement in the various projects that teams are running.

Reviewer: ```suggestion
This is measured using the Community Dashboard by reviewing team activity and engagement in the various projects the teams are managing.
```

-------------------------------------------

Become a corporate member of Mautic
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. vale off 

The first step is to start financially contributing to Mautic as a Corporate Member if you're not already. To do this you can sign up on the :xref:`Mautic Open Collective`. Be sure to switch to your organization account before you proceed with payment, so that it's associated with your company.

.. vale on 

Become a consistent contributor
-------------------------------

Once you have started contributing financially, take a look at where you are currently contributing to Mautic.

.. vale off 

We need every kind of skill from development to marketers, documentation writers, project managers, technical writers, designers - basically you bring the skills and we will find a place to put them to great use.

.. vale on

.. vale off 

After you have been actively contributing in both financial and practical terms to Mautic for at least 3 months as an organization, please fill out :xref:`this form`.

.. vale on

Be sure to include  the names of team members to allow for verification of contribution history..

.. vale off 

>  The application will be reviewed, and a response will be provided as soon as possible.

Reviewer 2: ```suggestion
 The application review takes a few weeks, with a and a response provided as soon as possible.
```
